sa/sa.go

```diff
@@ -887,7 +887,20 @@ func (ssa *SQLStorageAuthority) RevokeAuthorizationsByDomain(ctx context.Context
 
 // AddCertificate stores an issued certificate and returns the digest as
 // a string, or an error if any occurred.
-func (ssa *SQLStorageAuthority) AddCertificate(ctx context.Context, certDER []byte, regID int64, ocspResponse []byte) (string, error) {
+func (ssa *SQLStorageAuthority) AddCertificate(
+	ctx context.Context,
+	certDER []byte,
+	regID int64,
+	ocspResponse []byte,
+	issued *time.Time) (string, error) {
+	// NOTE(@cpu): Historically `AddCertificate` was hardcoded to set the added
```

Comment:
Can you file an issue to remove the default and error on nil time inputs, once this is deployed?

Reply:
Sure! https://github.com/letsencrypt/boulder/issues/3657